subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -33,6 +33,9 @@ import static org.gradle.work.ChangeType.REMOVED
 class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec implements ValidationMessageChecker {
     def setup() {
         expectReindentedValidationMessage()
+        // This is required for the daemon to clean up symlinks between builds
+        executer.requireDaemon()
+        executer.requireIsolatedDaemons()
```

Comment:
🤔 Why do you need isolated daemons, aren't daemons enough?

Reply:
I'm not 100% sure, but this is only temporary. It will be reverted (along with a few other places in other symlink-related integration tests) once https://github.com/gradle/gradle/issues/28008 is implemented.